contrib/babelfishpg_common/src/varchar.c

```diff
@@ -1102,6 +1250,109 @@ varchar2numeric(PG_FUNCTION_ARGS)
 	PG_RETURN_NUMERIC(result);
 }
 
+/*
+ * fixeddecimal2str_helper
+ * Converts a fixed decimal value to a string, rounding it off to 2 decimal points.
+ */
+#define FIXEDDECIMAL_MULTIPLIER_NEW 100LL
+#define FIXEDDECIMAL_SCALE_NEW 2
+static char *
+fixeddecimal2str_helper(int64 val, char *buffer)
```

Comment:
I understand that this is copied from fixeddecimal code, can we somehow, conditionally, do it over there? Would the create extension dependency chain be a deadlock in that case since varchar wont exist?

Reply:
Yep, we create fixeddecimal before sys.varchar so can't move this logic there.